manage_user_page.php

```diff
@@ -185,6 +190,17 @@
 $t_where_params = array();
 if( $f_filter === 'ALL' ) {
 	$t_where = '(1 = 1)';
+} else if( $f_findname<>"" ) {
+	$c_prefix = strtoupper(db_prepare_string($f_filter));
```

Comment:
function db_prepare_string is deprecated and should not be used any longer, see https://github.com/mantisbt/mantisbt/blob/release-2.16.0/core/database_api.php#L620